devbox/common/src/InitialScan.scala

```diff
@@ -12,7 +14,9 @@ object InitialScan {
 
       val fileStream = os.walk.stream.attrs(
         base,
-        (p, attrs) => skipper.initialScanIsPathSkipped(base, p.subRelativeTo(base), attrs.isDir)
+        (p, attrs) =>
+          syncIgnore.fold(false)(regex => regex.matches(p.relativeTo(base).toString())) ||
+          skipper.initialScanIsPathSkipped(base, p.subRelativeTo(base), attrs.isDir)
```

Comment:
Would it make sense to move this inside Skippers? That way all logic related to what files to sync (or not) is in a single place, and it's also easier to keep in sync the initial scan with the subsequent file system events (I think the change in SkipScanActor does that, but I'm not 100% sure).

Reply:
Possibly? If we allowed --sync-ignore to be configured on a per-repo basis then we'd need to that, but for now it's kind of a global flag